pkg/storage/utils/eosfs/eosfs.go

```diff
@@ -427,29 +452,63 @@ func (fs *eosfs) AddGrant(ctx context.Context, ref *provider.Reference, g *provi
 
 	fn := fs.wrap(ctx, p)
 
```

Comment:
Add here an if condition that behaves like this:

IF Grant is Deny:
    then do the logic below with `setAcls`
 ELSE 
     do the logic as before with `AddACL`

Reply:
@gmgigi96 can you add this logic?

Reply:
do we need it? for now is only for testing, when the --priority functionality will be implemented in EOS, we will use directly and only AddACL

Reply:
Yes we need it like that we can merge this PR and not have a big one.
I've started already to add, will push by end of day. 
I take care of this one

Reply:
And what about `GetOwners()`? Do we merge it as is for now?

Reply:
@labkode ok, didn't know i wanted to merge right now :)